packages/yew/src/dom_bundle/mod.rs

```diff
@@ -18,6 +18,7 @@ mod braw;
 mod bsuspense;
 mod btag;
 mod btext;
+mod dom_position;
```

Comment:
```suggestion
mod dom_slot;
```

This module should also be renamed.

Reply:
While it makes sense for the struct, since it only keeps track of the point in the childlist, not the parent, I don't see the same need for the module, I think it's even clearer as `dom_position`.

Reply:
In this case, I think it's better to name the module without the `dom_` prefix (`mod position;`) as it is no longer specific to a type with the same name and `dom_` is not repeated in the path.

Reply:
Agreed, just `position` is even better! I've also added a short module summary on top.